### DISCUSSION TITLE
Unable to access the chat section after setting it up manually

### DISCUSSION THREAD

**Discussion by nk8291** [RamiAwar/dataline#349] [Q&A]
@RamiAwar  I have setup the application for learning purpose on my local machine using github. I have followed the steps given in frontend and backend. after starting the application and accessing it on 5173 port, I am getting same content as the dataline.app website but not the actual configuration section where I can configure the data source and use ti for Question and answer purpose.

**Comment by RamiAwar** [comment]
Sorry about that 😓 @nk8291 

We have a weird setup where if you don't pass an environment variable to the frontend, then the "landing page" is what you see.
Check out the frontend setup docs, you need to pass in an environment variable called NODE_ENV!

Do you think we can bring attention to it in a better way in the docs? You seem to have missed it.

**Comment by nk8291** [comment]
Hi @RamiAwar , Thank you for your quick response. I have seen below option in readme document:

export NODE_ENV=local

did this setup.. my backend is also running on 7377 port.
Could you please help if anything is wrong from my end. I am bit new to node apps and in learning phase.

thank you for your assistance.

**Reply by RamiAwar** [reply]
You need to make sure you run the export in the same terminal as the frontend, and run the frontend after you run the export.

This should work, not sure why it doesn't? Did you refresh the browser page?

**Reply by RamiAwar** [reply]
Do you see an error in your backend process? We just released a new version that fixed a big bug, so maybe updating to latest helps! LMK

**Comment by nk8291** [comment]
![image](https://github.com/user-attachments/assets/da60c937-7e63-4ab9-a257-bad2fce977d2)
getting these errors.. I am using github repo

**Reply by nk8291** [reply]
still

![image](https://github.com/user-attachments/assets/c1d25bd7-d5ee-4223-a009-fee8892d1c65)

**Reply by nk8291** [reply]
any Idea why this is coming .. I have enabled CORS all ports @RamiAwar

**Reply by RamiAwar** [reply]
Why does it say something about port 5004? Did you configure it in a special way?

**Reply by RamiAwar** [reply]
Just use docker-compose, then you can develop locally and not worry about these things. Try following the instructions for that and keep me updated!
